### PR TITLE
kew: init at 1.5.2

### PR DESCRIPTION
**Pull request by Demine0**
kew: init at 1.5.2
homepage: https://github.com/ravachol/kew
Listen to music in the terminal.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/by-name/ke/kew/package.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+, fetchFromGitHub
+, ffmpeg
+, fftwFloat
+, chafa
+, freeimage
+, glib
+, pkg-config
+}:
+
+stdenv.mkDerivation {
```
Comment (azahi) [inline pkgs/by-name/ke/kew/package.nix:12]:
```suggestion
stdenv.mkDerivation rec {
```

File: pkgs/by-name/ke/kew/package.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  pname = "kew";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "ravachol";
+    repo = "kew";
+    rev = "v1.5.2";
```
Comment (azahi) [inline pkgs/by-name/ke/kew/package.nix:19]:
```suggestion
    rev = "v${version}";
```

File: pkgs/by-name/ke/kew/package.nix [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    hash = "sha256-Om7v8eTlYxXQYf1MG+L0I5ICQ2LS7onouhPGosuK8NM=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ ffmpeg freeimage fftwFloat.dev chafa glib.dev ];
```
Comment (azahi) [inline pkgs/by-name/ke/kew/package.nix:24]:
```suggestion
  buildInputs = [ ffmpeg freeimage fftwFloat chafa glib ];
```

File: pkgs/by-name/ke/kew/package.nix [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    "PREFIX=${placeholder "out"}/bin"
+  ];
+
+  meta = {
+    description = "kew is a command-line music player for Linux.";
+    homepage = "https://github.com/ravachol/kew";
+    platforms = lib.platforms.linux;
+    license = lib.licenses.gpl2Only;
+    maintainers = with lib.maintainers; [ demine ];
+  };
```
Comment (azahi) [inline pkgs/by-name/ke/kew/package.nix:37]:
```suggestion
  meta = with lib; {
    description = "A command-line music player for Linux";
    homepage = "https://github.com/ravachol/kew";
    platforms = platforms.linux;
    license = licenses.gpl2Only;
    maintainers = with maintainers; [ demine ];
  };
```

**Review by azahi — CHANGES_REQUESTED** [review]

**Comment by cab404** [comment]
It installs into `/bin/bin/kew`~

File: pkgs/by-name/ke/kew/package.nix [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    rev = "v1.5.2";
+    hash = "sha256-Om7v8eTlYxXQYf1MG+L0I5ICQ2LS7onouhPGosuK8NM=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ ffmpeg freeimage fftwFloat.dev chafa glib.dev ];
+
+  installFlags = [
+    "MAN_DIR=${placeholder "out"}/share/man"
+    "PREFIX=${placeholder "out"}/bin"
```
Comment (cab404) [inline pkgs/by-name/ke/kew/package.nix:28]: That should probably be just `${placeholder "out"}`

**Review by azahi — APPROVED** [review]

**Comment by Demine0** [comment]
Result of `nixpkgs-review pr 267234` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>kew</li>
  </ul>
</details>

**Pull request merged**